style.css

```diff
@@ -472,6 +472,138 @@ hr {
   display: none;
 }
 
+/* Pop-Up */
+.model_content {
+  width: 95%;
+  height: calc(100vh - 20px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 8px;
+  padding-bottom: 10px;
+  padding-top: 25px;
+  margin: 12px auto;
+}
+
+.model_header {
+  display: flex;
+  height: 20px;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin: 12px 0;
+}
```

Comment:
- [ ] The `see source` button is overflowing from the page bottom, Please fix this button positioning.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/215792518-e541b0f4-40ea-4491-9d41-2494360eafca.png)  | ![image](https://user-images.githubusercontent.com/68796232/215792481-3e683b56-2496-4f73-af9f-ded73691c67d.png) |